readme.md

```diff
@@ -225,3 +225,27 @@ As in: OOPs I used Java!
 **A:** Because he used up all his cache
 
 ---
+
+**Q:** What did the programmer say after he was fired because of bad practice?
+
+**A:** He refused to comment
+
+---
+
+**Q:** What do they call the principal of Git school?
+
+**A:** ``HEAD --> master ``
+
+---
+
+**Q:** Asynchronous JavaScript is amazing.
+
+**A:** I Promise you.
+
+---
+
+I never trust Matlab developers,
+
+they're always plotting something.
```

Comment:
This one is missing Q/A

Reply:
Can you submit a new PR with just the matlab joke? Then we can close this PR.

Reply:
I only left the matlab joke in.